nixos/modules/services/databases/lldap.nix

```diff
@@ -31,6 +31,30 @@ in
       '';
     };
 
+    user = mkOption {
+      type = types.str;
+      default = "lldap";
+      description = lib.mdDoc ''
+        User under which lldap should run (default lldap)
```

Comment:
These descriptions should end with a dot (full sentences).

Reply:
fixed